### PR TITLE
[IMP] Add EAN-14 & Fix GTIN

### PR DESCRIPTION
**Pull request by lasley**
* Add EAN-14 encoding
* Switch GTIN to EAN-14 - standard definition is a 14 digit number, not 13

Note that the GTIN switch is obviously API breaking. I'm indifferent on whether that part needs to be changed or not, just in the interest of not breaking things. [True definition of a GTIN is 14 digits though](http://www.gtin.info/):

> GTIN describes a family of GS1 (EAN.UCC) global data structures that employ 14 digits and can be encoded into various types of data carriers.

### CODE REVIEW

File: barcode/__init__.py [hunk 1]
```diff
@@ -54,7 +54,8 @@
     ean8=EAN8,
     ean13=EAN13,
     ean=EAN13,
-    gtin=EAN13,
+    ean14=EAN14,
+    gtin=EAN14,
```
Comment (kxepal) [inline barcode/__init__.py]: `gtin` was changed to `EAN14` with some reason?

Comment (kxepal) [inline barcode/__init__.py]: NVM, I miss the commit message.

**Comment by kxepal** [comment]
Thanks!

**Comment by lasley** [comment]
No, thank you! Drop a line if you need help on repo maintenance. I went ahead and watched it too so I can help out with reviews if some PRs come in.

**Pull request merged**